src/test/scala/dibl/StitchesSpec.scala

```diff
@@ -15,42 +15,105 @@
 */
 package dibl
 
+import org.scalatest.prop.TableDrivenPropertyChecks
 import org.scalatest.{FlatSpec, Matchers}
 
-class StitchesSpec extends FlatSpec with Matchers {
+class StitchesSpec extends FlatSpec with Matchers with TableDrivenPropertyChecks {
 
-  val allClothStitches = Array(
-    Array("ctc", "ctc", "ctc"),
-    Array("ctc", "ctc", "ctc")
-  )
-  "Settings.toStitchMatrix" should "default to cloth stitches" in {
-    Settings.toStitchMatrix("", 2, 3) shouldBe allClothStitches
-  }
-  it should "apply a custom default and be case insensitive" in {
-    Settings.toStitchMatrix("tc a2=ctc B1=TCtc", 2, 3) shouldBe Array(
-      Array("tc", "tctc", "tc"),
-      Array("ctc", "tc", "tc")
-    )
-  }
-  it should "ignore an invalid default stitch" in {
-    Settings.toStitchMatrix("p", 2, 3) shouldBe allClothStitches
-  }
-  it should "ignore a node id without a stitch" in {
-    Settings.toStitchMatrix("A1=", 2, 3) shouldBe allClothStitches
-  }
-  it should "ignore a stitch with just a pin" in {
-    Settings.toStitchMatrix("A1=p", 2, 3) shouldBe allClothStitches
-  }
-  it should "ignore an invalid stitch" in {
-    Settings.toStitchMatrix("A1=.", 2, 3) shouldBe allClothStitches
-  }
-  it should "ignore a node with the column out of range" in {
-    Settings.toStitchMatrix("D1=tc", 2, 3) shouldBe allClothStitches
-  }
-  it should "ignore a node with the row out of range" in {
-    Settings.toStitchMatrix("a22=tc", 2, 3) shouldBe allClothStitches
+  "the stitch at matrix position A1" should "have expected color" in {
+
+    forAll(Table("input" -> "color",
+
+      // ordinary cases
+      "" -> "purple", // default is ctc
+      "t" -> "",
+      "ct" -> "green",
+      "tc" -> "green",
+      "tct" -> "green",
+      "ctc" -> "purple",
+      "ctct" -> "red",
+      "tctc" -> "red",
+      "tctct" -> "red",
+      "ttctc" -> "red",
+      "ctctt" -> "red",
+      "cttc" -> "turquoise",
+      "tcttc" -> "turquoise",
+      "cttct" -> "turquoise",
+      "cltc" -> "brown",
+      "crtc" -> "brown",
+      "crrc" -> "brown",
+      "clllc" -> "brown",
+      "ctttc" -> "brown",
+      "ctctc" -> "blue",
+      "ctctct" -> "blue",
+      "tctctc" -> "blue",
+      "ctctctc" -> "blue",
```

Comment:
Weird that this test succeeds but it does indeed not show  up in the diagrams.

Reply:
Ah, it's missing here:
https://github.com/jo-pol/GroundForge/blob/master/src/main/scala/dibl/SVG.scala#L104-L113